src/kmeans.jl

```diff
@@ -349,9 +362,8 @@ function repick_unused_centers(
         j = wsample(1:n, tcosts)
         tcosts[j] = 0
         v = view(X, :, j)
-        centers[:, i] = v
-
-        colwise!(ds, SqEuclidean(), v, X)
+        centers[:, i] .= v
```

Comment:
Is `.=` really needed? If not, better use `=`, which is stricter.

Reply:
ok